.github/issue_template.md

```diff
@@ -0,0 +1,21 @@
+**Please note:** This repository's issues are reserved for feature requests and bug reports.
```

Comment:
is it, though? i can think of lots of issues that wouldn't cover, but that we still want to know about because it means we can improve the docs 🤔

Reply:
My ideal would be that people use Stack Overflow for asking for implementation help, and therefore hopefully get help from other users.  Then that frees us/our issues list up for genuine bugs and requests (including requests for more clear documentation if something keeps coming up in SO).

But this is definitely a stronger push than what we've done in the past.  I'm chucking this out there to see what you folks think, if we did want to have a stricter line like some OSS projects have. Open question as to whether we'd actually enforce this, or if we just use it as a nudge to get some people to go to SO and we can still answer requests that end up here.